GIT-VERSION-GEN

```diff
@@ -60,21 +74,21 @@ else
 	VN="$DEF_VER"
 fi
 
-dirty=$(sh -c 'git diff-index --name-only HEAD' 2>/dev/null) || dirty=
+dirty=$(sh -c "git -C '$SOURCE_DIR' diff-index --name-only HEAD" 2>/dev/null) || dirty=
 case "$dirty" in
 '')
 	;;
 *)
 	VN="$VN-dirty" ;;
 esac
 
-if test -r $GVF
+if test -r $OUTPUT
 then
-	VC=$(sed -e 's/^GITGUI_VERSION = //' <$GVF)
+	VC=$(sed -e 's/^GITGUI_VERSION = //' <$OUTPUT)
 else
 	VC=unset
 fi
 test "$VN" = "$VC" || {
 	echo >&2 "GITGUI_VERSION = $VN"
-	echo "GITGUI_VERSION = $VN" >$GVF
+	echo "GITGUI_VERSION = $VN" >$OUTPUT
 }
```

Comment:
This would be a good time to put all `"$OUTPUT"` in double-quotes.

Reply:
Yup, makes sense indeed.